src/lib/openapi/spec/project-schema.ts

```diff
@@ -89,6 +89,71 @@ export const projectSchema = {
             description:
                 'The average time from when a feature was created to when it was enabled in the "production" environment during the current window',
         },
+        owners: {
+            description: 'The owners of this project, both users and groups.',
+            oneOf: [
+                {
+                    type: 'array',
+                    items: {
+                        anyOf: [
+                            {
+                                type: 'object',
+                                required: ['ownerType', 'name'],
+                                properties: {
+                                    ownerType: {
+                                        type: 'string',
+                                        enum: ['user'],
+                                    },
+                                    name: {
+                                        type: 'string',
+                                        example: 'User Name',
+                                    },
+                                    imageUrl: {
+                                        type: 'string',
+                                        nullable: true,
+                                        example:
+                                            'https://example.com/image.jpg',
+                                    },
+                                    email: {
+                                        type: 'string',
+                                        nullable: true,
+                                        example: 'user@example.com',
+                                    },
+                                },
+                            },
+                            {
+                                type: 'object',
+                                required: ['ownerType', 'name'],
+                                properties: {
+                                    ownerType: {
+                                        type: 'string',
+                                        enum: ['group'],
+                                    },
+                                    name: {
+                                        type: 'string',
+                                        nullable: true,
+                                        example: 'Group Name',
+                                    },
+                                },
+                            },
+                        ],
+                    },
+                },
+                {
+                    type: 'array',
```

Comment:
We can specify the length here. Or we could maybe add an enum? That'd be ideal, but I'm not sure what works.

```suggestion
                    type: 'array',
                    minItems: 1,
                    maxItems: 1,
                    enum: ['{ownerType: 'system'}]
```

Reply:
I'll try, but I had issues with this failing before. OpenAPI didn't handle oneOf + maxItems in a correct way

Reply:
Interesting. Not sure what the deal is. My preference would be to just set the type to array and provide the enum, but I'm not sure how it handles that